settings/dev.settings.json

```diff
@@ -2,10 +2,10 @@
   "ROOT_URL": "",
   "MAIL_URL": "",
   "reaction": {
-    "REACTION_USER": "",
-    "REACTION_AUTH": "",
-    "REACTION_EMAIL": ""
+    "REACTION_USER": "flevian",
```

Comment:
Is this right? Seems like merging this will result to anyone who pulls develop using your admin details.